examples/01a-Getting-started.ipynb

```diff
@@ -0,0 +1,632 @@
+{
```

Comment:
<p>Lets remove this. This is pretty confusing :) We would not validate without training?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>sure thing, removed</p><p> </p>